velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -54,7 +54,9 @@ class PlanBuilder {
   explicit PlanBuilder(memory::MemoryPool* pool = nullptr)
       : PlanBuilder(std::make_shared<PlanNodeIdGenerator>(), pool) {}
 
-  PlanBuilder& tableScan(const RowTypePtr& outputType);
```

Comment:
This is a nice change, but feels unrelated. Would it make sense to move it into a separate PR?